ping/rust/Dockerfile

```diff
@@ -11,7 +11,7 @@ ARG GIT_TARGET=""
 RUN if [ ! -z "${GIT_TARGET}" ]; then sed -i "s,^git.*,git = \"https://${GIT_TARGET}\"," ./plan/Cargo.toml; fi
 
 ARG GIT_REF=""
-RUN if [ ! -z "${GIT_REF}" ]; then sed -i "s/^rev.*/rev = \"${GIT_REF}\"/" ./plan/Cargo.toml; fi
+RUN if [ ! -z "${GIT_REF}" ]; then sed -i "s/^rev.*/branch = \"${GIT_REF}\"/" ./plan/Cargo.toml; fi
```

Comment:
This change breaks the `custom` build, as that one uses a rev and not a branch :man_facepalming: silly me.

Reply:
`rev` should accept a branch name actually: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

Reply:
Have you tried `origin/master`?

Reply:
You might need to prefix with `refs/`:

- `refs/remotes/origin/master`
- `refs/heads/master`

Reply:
> `rev` should accept a branch name actually: [doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)

This is talking about named references being supported so I think one of those two should work.

Reply:
Wonderful. Thanks @thomaseizinger. `refs/heads/master` resolves to the current master commit (https://github.com/libp2p/rust-libp2p/commit/babacc5555e8c6d2fdd25ad41a8654548870d2a9). For some reason `refs/removes/origin/master` points to https://github.com/libp2p/rust-libp2p/commit/b99a2137ac12a295808c08931422dda86b52284a, a previous commit on master.

I created https://github.com/libp2p/test-plans/pull/81/ adopting the former. Please take a look.